api/routes/users.js

```diff
@@ -1,9 +1,16 @@
 var express = require('express');
 var router = express.Router();
+const QueryPromise = require("../database/DBService");
 
 /* GET users listing. */
 router.get('/', function(req, res, next) {
-  res.send('respond with a resource');
+  QueryPromise("SELECT * FROM Users")
+    .then((results) => {
+      res.send(results);
+    })
+    .catch((err) => {
+      res.send(err);
+    });
```

Comment:
Minor since it'll get changed later, but this seems unnecessary to have in this function?

Reply:
Yes, will comment this out later.

Reply:
Ok cool!